org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/ecore/EMFGeneratorFragment2.xtend

```diff
@@ -205,7 +206,7 @@ class EMFGeneratorFragment2 extends AbstractXtextGeneratorFragment {
 	
 	/**
 	 * Sets the BaseClass for the EClasses in the inferred GenModel.
-	 * Default value is {@link org.eclipse.emf.ecore.impl.MinimalEObjectImpl$Container}.
+	 * Default value is {@link MinimalEObjectImpl.Container}.
```

Comment:
The JavaDoc for inner classes needs a to be fully qualified (but '.' instead of '$')

Reply:
wonder why jdt does that

Reply:
or xtend

Reply:
Most likely a bug in Xtend